src/cgame/cg_rocket.cpp

```diff
@@ -642,3 +639,29 @@ bool CG_Rocket_IsCommandAllowed( rocketElementType_t type )
 			return false;
 	}
 }
+
+bool CG_Rocket_LoadCursor( Str::StringRef cursorPath )
+{
+	if ( cursorPath.empty() )
+	{
+		rocketInfo.cursor = 0;
+		return true;
+	}
+	rocketInfo.cursor = trap_R_RegisterShader( cursorPath.c_str(), (RegisterShaderFlags_t) RSF_DEFAULT );
+	if ( rocketInfo.cursor == 0 )
+	{
+		return false;
+	}
+	// Scale cursor with resolution while maintaining the original aspect ratio.
+	int x, y;
+	trap_R_GetTextureSize( rocketInfo.cursor, &x, &y );
+	float ratio = static_cast<float>( x ) / static_cast<float>( y );
```

Comment:
This reminds me a TODO I kept in another PR :) a single cast would suffice (not important at all though).